presto-redshift/src/main/java/com/facebook/presto/plugin/redshift/RedshiftClient.java

```diff
@@ -66,4 +70,16 @@ protected void renameTable(JdbcIdentity identity, String catalogName, SchemaTabl
             throw new PrestoException(JDBC_ERROR, e);
         }
     }
+
+    @Override
+    public Optional<ReadMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
+    {
+        String typeName = typeHandle.getJdbcTypeName();
+
+        if (typeName.equalsIgnoreCase("binary varying")) {
```

Comment:
Can you add a comment explaining that this is dependent on the version of the driver, as is included in this PR?